opencti-platform/opencti-front/src/private/components/common/stix_core_relationships/views/EntityStixCoreRelationshipsRelationshipsView.tsx

```diff
@@ -143,7 +143,18 @@ const EntityStixCoreRelationshipsRelationshipsView: FunctionComponent<EntityStix
   }
   const contextFilters: FilterGroup = {
     mode: 'and',
-    filters: predefinedFilters,
+    filters: [
+      ...predefinedFilters,
+      {
+        key: 'regardingOf',
+        operator: 'eq',
+        mode: 'and',
+        values: [
+          { key: 'id', values: [entityId], operator: 'eq', mode: 'or' },
+          { key: 'relationship_type', values: relationshipTypes, operator: 'eq', mode: 'or' },
+        ] as unknown as string[], // Workaround for typescript waiting for better solution
```

Comment:
This means we have to define different cases for values depending on the key.
I can have a look if you need help.

Reply:
'in regards of' is a filter for entities, not relationships